### ISSUE TITLE
TypeError when file can't be found on start and then on every keypress in that tab

### ISSUE THREAD

**Issue by AgentME** [AtomLinter/linter-flow#63]
I got this error message in a red notification when I started Atom. When I last started Atom in this directory, I had a file named "foo.js" opened, so Atom started up with a tab for that file, but the file no longer existed. I can reproduce the error by repeating these steps. Then, if you type in the tab, this error fires again on every keypress.

```
TypeError: Cannot read property 'indexOf' of null
    at Object.lint (/Users/chris/.atom/packages/linter-flow/lib/index.js:81:21)
    at promises.push.Promise.then._this.emitter.emit.linter (/Users/chris/.atom/packages/linter/lib/linter-registry.js:63:26)
    at /Users/chris/.atom/packages/linter/lib/linter-registry.js:62:23
    at Set.forEach (native)
    at LinterRegistry.lint (/Users/chris/.atom/packages/linter/lib/linter-registry.js:60:18)
    at /Users/chris/.atom/packages/linter/lib/linter.coffee:98:16
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at /Users/chris/.atom/packages/linter/lib/editor-linter.js:80:22
    at /Users/chris/.atom/packages/linter/lib/helpers.coffee:23:18
```

**Comment by nmn** [comment]
should be fixed in #80
Though I haven't been able to reproduce it.

**Comment by nmn** [comment]
Fixed in 5.1.0
